src/javatests/com/google/cloud/bigquery/dwhassessment/extractiontool/dbscripts/InternalScriptLoaderTest.java

```diff
@@ -444,14 +491,45 @@ public void loadScripts_queryLogs_with_timeRange() throws IOException, SQLExcept
             .set("TotalFirstRespTime", 1234.5)
             .set("TotalIOCount", 1234.56)
             .build();
+    GenericRecord expectedQueryLogsRecord2 =
```

Comment:
To keep tests focused, let's move record construction to a method named like `getQueryLogsRecordBuilder` and redefine only values required for the test ("QueryText") in a test body.

Reply:
It's a bit unfortunate that part of the setup is actually in a separate .sql data file. I agree because of the sheer line numbers needed to define an expected record, it's becoming too DAMP even for the tests.
Changed to keep the most relevant information available without diffusing the readers' attention.